sql-aggregates/movie-maniacs.sql

```diff
@@ -0,0 +1,9 @@
+select
+"customers"."firstName",
+"customers"."lastName",
+sum("amount") as "totalSpent"
+from "payments"
+join "customers" using ("customerId")
+join "rentals" using ("rentalId")
+group by "customers"."firstName", "customers"."lastName"
```

Comment:
It would be best to group by customerId to cover the case where two customers have the exact same first and last names.